<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="left_frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="window_label">
        <property name="text">
         <string>Window</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="window_box"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="windowtext_label">
        <property name="text">
         <string>WindowText</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="windowtext_box"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="button_label">
        <property name="text">
         <string>Button</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="button_box"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="buttontext_label">
        <property name="text">
         <string>ButtonText</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="buttontext_box"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="base_label">
        <property name="text">
         <string>Base</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="base_box"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="right_frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="type_label">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="type_box"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="text_label">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="text_edit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextEdit" name="text_demo"/>
      </item>
      <item row="3" column="1">
       <widget class="QDialogButtonBox" name="btns_box">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
